pkgs/development/interpreters/php/generic.nix

```
@@ -387,6 +387,10 @@ let
               in
               php;
             inherit ztsSupport;
+
+            services.default = {
+              imports = [ ./service.nix ];
+            };
```

It'd be weird for the `php81.services.default` (an older release) to have the current `php` as its default.
```suggestion
              php-fpm.package = lib.mkDefault finalAttrs.finalPackage;
            };
```

`finalAttrs` can be acquired as `mkDerivation (finalAttrs: ...)` above.

---

@drupol i seem to recall you were a big fan of the `finalAttrs` pattern - is there any blockers you're aware of for us to utilize `finalAttrs` in the `php` packages?